tests/test_serializers/test_json.py

```diff
@@ -18,6 +18,7 @@ class TestNumericEncoding:
 
     NUMERIC_DECODING_TEST_CASES = [
         (3 + 4j, CustomEncodedNumericTypes.complex.json_encoding([3, 4])),
+        (Decimal(3.1415), CustomEncodedNumericTypes.decimal.json_encoding(3.1415)),
```

Comment:
Having `Decimal("3.1415")` (with number quoted) causes `TestNumericEncoding.test_decode_numeric` to fail.